src/providers/uploadsManager/uploadsManager.tsx

```diff
@@ -23,20 +23,10 @@ export const UploadsManager: React.FC = () => {
     (state) => state.uploads.filter((asset) => asset.owner === activeChannelId),
     shallow
   )
-
-  const { addAsset, removeAsset, setIsSyncing, isSyncing, pendingAssetsIds, removePendingAssetId, setUploadStatus } =
-    useUploadsStore(
-      (state) => ({
-        addAsset: state.addAsset,
-        removeAsset: state.removeAsset,
-        setIsSyncing: state.setIsSyncing,
-        isSyncing: state.isSyncing,
-        pendingAssetsIds: state.pendingAssetsIds,
-        removePendingAssetId: state.removePendingAssetId,
-        setUploadStatus: state.setUploadStatus,
-      }),
-      shallow
-    )
+  const { addAsset, removeAsset, setIsSyncing, removePendingAssetId, setUploadStatus } = useUploadsStore(
+    (state) => state.actions
+  )
+  const { isSyncing, pendingAssetsIds } = useUploadsStore((state) => state)
```

Comment:
This will slightly change the behaviour. Previously we provided an object extracted from state and asked for a `shallow` comparison. That meant that only if one of the requested items changed, this component would rerender. With `useUploadsStore((state) => state)`, the component will rerender on any change in the `state`, are we sure that is what we want?